ui/apps/platform/src/Containers/PolicyCategories/PolicyCategorySidePanel.tsx

```diff
@@ -0,0 +1,108 @@
+    addToast: (toast) => void;
+};
+
+function PolicyCategorySidePanel({
+    selectedCategory,
+    setSelectedCategory,
+    addToast,
+}: PolicyCategoriesSidePanelProps) {
+    const formik = useFormik({
+        initialValues: selectedCategory,
+        onSubmit: (values, { setSubmitting }) => {
+            console.log('onSumbit');
+            setSubmitting(false);
+            const { id, name } = values;
+            renamePolicyCategory(id, name)
+                .then(() => {
+                    setSelectedCategory(values);
+                })
+                .catch((error) => {
+                    addToast(error.message);
+                })
+                .finally(() => {
+                    setSubmitting(false);
+                });
+        },
+    });
+
+    const { values, handleChange, dirty, handleSubmit } = formik;
+
+    function onChange(_value, event) {
+        handleChange(event);
+    }
+
+    function clearSelectedCategory() {
+        setSelectedCategory();
+    }
+
+    const { name } = selectedCategory;
+
+    return (
+        <>
+            <PageSection isFilled variant="light" className="pf-u-h-100">
+                <Flex direction={{ default: 'column' }} className="pf-u-h-100">
+                    <Flex
+                        justifyContent={{ default: 'justifyContentSpaceBetween' }}
+                        fullWidth={{ default: 'fullWidth' }}
+                        flexWrap={{ default: 'nowrap' }}
+                    >
+                        <Title headingLevel="h3">{name}</Title>
+                        <Button variant="secondary" isDanger>
+                            Delete category
+                        </Button>
+                    </Flex>
+                    <FormikProvider value={formik}>
+                        <Form>
+                            <FormGroup
+                                fieldId="policy-category-name"
+                                label="Category name"
+                                isRequired
+                                helperText="Provide a descriptive and unique category name."
+                            >
+                                <TextInput
+                                    id="name"
+                                    type="text"
+                                    value={values.name}
+                                    onChange={onChange}
+                                />
+                            </FormGroup>
+                            <ActionGroup>
+                                <Button
+                                    variant="primary"
+                                    isDisabled={!dirty}
+                                    onClick={() => handleSubmit()}
+                                >
+                                    Save
+                                </Button>
+                                <Button variant="secondary" onClick={clearSelectedCategory}>
+                                    Cancel
```

Comment:
A thought, although not sure whether it is a good thought, is whether text depends on whether or not edited?

Cancel versus Close? Somehow, it caused me to pause and think twice how to close the panel without changes.